### ISSUE TITLE
Errors with GXL schema

### ISSUE THREAD

**Issue by felixfbecker** [angelozerr/lsp4xml#264]
I configured vscode-xml to use the GXL schema http://www.gupro.de/GXL/xmlschema/gxl-1.0.xsd for all `.gxl` files.

But when opening a GXL file, it gives a lot of errors:

![image](https://user-images.githubusercontent.com/10532611/49963297-d0f75180-ff17-11e8-9d23-622455250cd8.png)

And none of the code intelligence features work.

Here's an example file: http://www.gupro.de/GXL/examples/instance/gxl/simpleExample/content.html

The schema is official so I would expect it to work (and assume it's correct)

**Comment by fbricon** [comment]
have you tried with the [latest](http://download.jboss.org/jbosstools/vscode-xml/staging/?C=M;O=D) vscode-xml ci build?

**Comment by felixfbecker** [comment]
No, I am using the marketplace version 0.2.0

**Comment by felixfbecker** [comment]
Same errors with vscode-xml-0.3.0-671.vsix

**Comment by fbricon** [comment]
have you succeeded in getting it to work in another IDE/editor? If yes, which one and how?

**Comment by felixfbecker** [comment]
No, this is the first time I use this and the first IDE I tried (and have available that I know supports this...)

**Comment by NikolasKomonen** [comment]
After a quick look, in the XSD it uses
`xmlns:xsd="http://www.w3.org/2000/10/XMLSchema"`
instead of the regular
`xmlns:xsd="http://www.w3.org/2001/XMLSchema"`

and from the errors it looks like it only works with the latter.

Not sure if xerces only supports the 2001 schema, need to investigate it.

**Comment by angelozerr** [comment]
I confirm what @NikolasKomonen said. I remember that I started to implement XSD validator but it was not finished. After activating XSD validator, I have this error:

![image](https://user-images.githubusercontent.com/1932211/57857189-e63a1a00-77ee-11e9-94b7-5f264a56cf38.png)

I will try to finish my work on XSD validator and I will activate it. It will help a lot after to understand problems coming from XML Schema.

**Comment by angelozerr** [comment]
I fear that issue cannot be resolved since this error comes from xerces here https://github.com/apache/xerces2-j/blob/trunk/src/org/apache/xerces/impl/xs/traversers/XSAttributeChecker.java#L988

```java
if (!SchemaSymbols.URI_SCHEMAFORSCHEMA.equals(uri)) {
  reportSchemaError("s4s-elt-schema-ns", new Object[] {elName}, element);
}
```
As you can see the test with namespace uri is hard coded.
